### ISSUE TITLE
go-charset not available as dependancy anymore

### ISSUE THREAD

**Issue by mihailgmihaylov** [Sjeanpierre/passenger-datadog-monitor#7]
Hello,

Today I tried to build the tool with:
`go get -v github.com/Sjeanpierre/passenger-datadog-monitor`
but get:
```
package code.google.com/p/go-charset/charset: unable to detect version control system for code.google.com/ path
package code.google.com/p/go-charset/data: unable to detect version control system for code.google.com/ path
```

I think to use this package as a dependancy from google code we need some kind of permissions:
https://code.google.com/archive/p/go-charset/downloads

This brakes the build of the project. Can you have a look?

**Comment by protoblade** [comment]
This can be easily fixed by using github version of the repo ->  https://github.com/rogpeppe/go-charset

**Comment by Sjeanpierre** [comment]
@mihailgmihaylov Thanks for opening this issue. I will correct this later today.

**Comment by onyxraven** [comment]
That go-charset repo says it is superceded. It looks like the replacement is https://godoc.org/golang.org/x/text/encoding - https://groups.google.com/d/msg/Golang-nuts/FHmzYmM5r5Y/0LMjMDlZCwAJ

**Comment by Sjeanpierre** [comment]
@mihailgmihaylov This issue should now be resolved with PR #10. please confirm that things are good now.
